### PR TITLE
Update busco

### PR DESCRIPTION
**Pull request by rlibouba**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by rlibouba** [comment]
Hello, I'm having a problem filling in the conditions for the miniprot output file to be generated. The mode selected must be genomic mode and the "Generate miniprot output" condition must be selected. 
When these conditions are met, the miniprot output file is generated and compliant. However, when one or both of these conditions are not selected, this output is generated but is empty. 
I've tried to correct this error but I'm stuck. Do you have any suggestions or explanations?
Thank you in advance. Have a nice day!

File: tools/busco/busco.xml [hunk 1]
```diff
@@ -178,7 +194,10 @@ busco
         <data name='busco_gff' format='gff3' label="${tool.name} on ${on_string}: GFF" from_work_dir="busco_output.gff">
             <filter>outputs and 'gff' in outputs</filter>
         </data>
+        <data name='busco_miniprot' format='gff3' label="${tool.name} on ${on_string}: Miniprot" from_work_dir="busco_galaxy/run_*/miniprot_output/*.gff">
+            <filter>busco_mode.mode == 'geno' and busco_mode.miniprot == true</filter>        </data>
```
Comment (bgruening) [inline tools/busco/busco.xml:198]:
```suggestion
        <data name='busco_miniprot' format='gff3' label="${tool.name} on ${on_string}: Miniprot" from_work_dir="busco_galaxy/run_*/miniprot_output/*.gff">
            <filter>busco_mode['mode'] == 'geno' and busco_mode['miniprot']</filter>
        </data>               
```

Can you try this?

**Comment by bgruening** [comment]
I think the tests are passing, its just the linting that needs to be fixed.

File: tools/busco/busco.xml [hunk 2]
```diff
@@ -269,6 +290,7 @@ busco
                     <has_text text="##gff-version 3" />
                 </assert_contents>
             </output>
+            <output name="busco_miniprot" file="genome_results/miniprot" compare="diff" lines_diff="4" />
```
Comment (bgruening) [inline tools/busco/busco.xml:293]: genome_results/miniprot is an empty file.
You probably want to compare against [genome_results/miniprot.gff](https://github.com/galaxyproject/tools-iuc/pull/5608/files#diff-d72f5f70cdc4f24da2fe63c2d7e7a5fea7c6f52da4bb8008057978392d32ada1)

**Comment by bgruening** [comment]
Let me know how I can help.

**Comment by rlibouba** [comment]
Hi, I think the update is ready to be merged

**Review by bgruening — APPROVED** [review]

**Review by bgruening — APPROVED** [review]

**Pull request merged**